src/libtest/lib.rs

```diff
@@ -1103,6 +1108,12 @@ fn get_concurrency() -> usize {
         }
     }
 
+    #[cfg(target_os = "redox")]
+    fn num_cpus() -> usize {
```

Comment:
Redox needs to provide a way to get the number of CPUs. We return `1` as a safe alternative